fuel-indexer/indexer/src/bin/service/main.rs

```diff
@@ -0,0 +1,77 @@
+use anyhow::Result;
+use async_std::{fs::File, io::ReadExt};
+use fuel_core::service::{Config, FuelService};
+use fuel_wasm_executor::{GraphQlAPI, IndexerConfig, IndexerService, Manifest};
+use log::info;
+use std::path::PathBuf;
+use structopt::StructOpt;
+use tokio::join;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "Indexer Service",
+    about = "Standalone binary for the fuel indexer service"
+)]
+pub struct Args {
+    #[structopt(short, long, help = "run local test node")]
+    local: bool,
+    #[structopt(parse(from_os_str), help = "Indexer service config file")]
+    config: PathBuf,
+    #[structopt(short, long, parse(from_os_str), help = "Indexer service config file")]
+    test_manifest: Option<PathBuf>,
+}
+
+#[tokio::main]
+pub async fn main() -> Result<()> {
+    env_logger::init();
```

Comment:
We prefer using tracing instead of env_logger: https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/logging.md